config/default/aad-pod-identity-deployment.yaml

```
@@ -63,7 +63,7 @@ metadata:
     k8s-app: capz-aad-pod-id-nmi-binding
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: manager
   namespace: capz-system
```

```suggestion
  namespace: system
```

the `manager` service account is created in `system` namespace

---

Fixed, thanks!